### DISCUSSION TITLE
Upgrading confusions

### DISCUSSION THREAD

**Discussion by mertcihankurel** [JedWatson/react-select#5257] [Q&A]
Recently I upgraded our react-select version from V2 to V5. Also, I updated the typescript version from 3.2 to 3.8. I'm getting this error related to the library in many places. Does anyone have an idea?
<img width="934" alt="Screen Shot 2022-07-05 at 15 58 24" src="https://user-images.githubusercontent.com/65771498/177333332-e9101fdd-b8d6-4f70-b5e7-2e273e8f8dd4.png">

**Comment by Methuselah96** [comment]
How is `rsProps` defined? And what version of `@types/react` are you using?
